### PR TITLE
fix: can create application

### PR DESCRIPTION
**Pull request by Linvery**
Fix: https://github.com/casdoor/casdoor-php-sdk/issues/30

### CODE REVIEW

**Comment by hsluoyz** [comment]
@Yamituki plz review

File: src/Auth/Application.php [hunk 1]
```diff
@@ -5,167 +5,124 @@
...
-    /**
-     * @var string
-     */
-    public $signinHtml;
-
-    public function __construct(string $owner, string $name)
+    public string $owner = "admin";
+    public string $name;
+    public string $createdTime;
+    public array $providers = [["name" => "provider_captcha_default", "canSignUp" => false, "canSignIn" => false, "canUnlink" => false, "prompted" => false, "alertType" => "None"]];
```
Comment (hsluoyz) [inline src/Auth/Application.php:18]: No need for all default values

File: src/Auth/Application.php [hunk 2]
```diff
@@ -5,167 +5,124 @@
 namespace Casdoor\Auth;
```
Comment (hsluoyz) [inline src/Auth/Application.php:5]: Apache license header for all PHP files

File: tests/ApplicationTest.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<?php
```
Comment (hsluoyz) [inline tests/ApplicationTest.php:1]: Apache license header for all PHP files

File: tests/ApplicationTest.php [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+namespace Casdoor\Tests;
+
+use Casdoor\Auth\Application;
+
+class ApplicationTest
+{
+    public function initConfig()
+    {
```
Comment (hsluoyz) [inline tests/ApplicationTest.php:10]: If the test us un-runnable, we usually don't add it. We can use example repo to verify it works

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: src/Auth/Application.php (hunk 1 above)
Comment (Linvery) [reply src/Auth/Application.php]: modified

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**